### DISCUSSION TITLE
关于火绒报毒问题

### DISCUSSION THREAD

**Discussion by WuMe-sicx** [jeessy2/ddns-go#1067] [Q&A] [answered]
病毒名称：Trojan/HTML.Phishing.ee
病毒ID：7796D4DAEC3274D9
病毒URL：http://172.17.1.6:9876/
操作结果：已阻止

进程ID：6624
操作进程：C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe
操作进程命令行："C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" --type=utility --utility-sub-type=network.mojom.NetworkService --lang=zh-CN --service-sandbox-type=none --use-angle=d3d11 --no-appcompat-clear --field-trial-handle=2468,i,6904168115550332173,6452087408544442049,262144 --variations-seed-version --mojo-platform-channel-handle=2624 /prefetch:3

不清楚什么原因,会概率触发调用的是ipv6 开启了webhook
版本6.3.1

**Comment by jeessy2** [comment, marked as answer]
降低版本

**Reply by WuMe-sicx** [reply]
我发现应该是Webhook 导致火绒报毒
http://172.17.1.6:9876 //开启了Webhook 

http://172.17.1.7:9876 //未开启Webhook
